integration_tests/testsuite/deploy_app.py

```diff
@@ -43,15 +47,18 @@ def deploy_app(image, appdir):
             fout.close()
         fin.close()
 
+        global DEPLOYED_VERSION
+        DEPLOYED_VERSION = test_util.generate_version()
+
         # TODO: once sdk driver is published, use it here
-        deploy_command = ['gcloud', 'app', 'deploy',
-                          '--stop-previous-version', '--verbosity=debug']
+        deploy_command = ['gcloud', 'app', 'deploy', '--version',
+                          DEPLOYED_VERSION, '--verbosity=debug']
 
         deploy_proc = subprocess.Popen(deploy_command,
                                        stdout=subprocess.PIPE,
                                        stdin=subprocess.PIPE)
 
-        output, error = deploy_proc.communicate()
+        output, _ = deploy_proc.communicate()
```

Comment:
the main reason I used `subprocess.Popen().communicate()` here is so that I can use `subprocess.PIPE` to say "yes" to the command when it asks "do you want to do this" automatically. check_output doesn't let you do that AFAIK